pilot/docker/envoy_pilot.json

```diff
@@ -3,6 +3,27 @@
    {
     "address": "tcp://0.0.0.0:15003",
     "name": "tcp_0.0.0.0_15003",
+    "filters": [
```

Comment:
How about making this ( and the next one ) http listeners, so we get proper stats ?